scripts/prepare_release.sh

```diff
@@ -49,7 +49,7 @@ function commit() {
   git commit -a -m "Data Release: $VERSION"
 }
 
-update_version
+# update_version
```

Comment:
revert?

Reply:
update_version was broken when autopush.datacommons.org needed login :(.  Commenting it makes rest of the flow work.